www/assets/js/components/SearchPage.js

```diff
@@ -229,9 +233,19 @@ export default function SearchPage() {
                     Resources
                   </button>
                 </li>
-                <li className="nav-item flex-grow-1 d-flex align-items-center justify-content-center results-total">
-                  {completedInitialLoad ? `${total} Results` : null}
-                </li>
+                {!isResourceSearch ? (
+                  <li className="sort-nav-item nav-item flex-grow-1 d-flex align-items-center justify-content-end">
+                    Sort By:{" "}
+                    <select
+                      value={serializeSort(sort)}
```

Comment:
I spun up `open-discussions` locally and ran:

```
gumaerc@gumaerc-laptop:~/Code/open-discussions$ docker-compose run --rm web ./manage.py recreate_index --course
/usr/lib/python3/dist-packages/requests/__init__.py:89: RequestsDependencyWarning: urllib3 (1.26.2) or chardet (3.0.4) doesn't match a supported version!
  warnings.warn("urllib3 ({}) or chardet ({}) doesn't match a supported "
Creating open-discussions_web_run ... done
Started celery task f85818f6-3697-4614-bfaa-e798d7045cad to index content for the following indexes: ['course']
Waiting on task...
Recreate index finished, took 0.236952 seconds
```
...and then spun up `ocw-hugo-themes/www` and went to `/search`  The page failed to load with these errors:
![image](https://user-images.githubusercontent.com/12089658/121968352-2ce8fe00-cd40-11eb-8777-fb2b30b15ad7.png)

```
SearchPage.js?fad4:240 Uncaught TypeError: Object(...) is not a function
    at SearchPage (SearchPage.js?fad4:240)
    at renderWithHooks (react-dom.development.js?61bb:14803)
    at mountIndeterminateComponent (react-dom.development.js?61bb:17482)
    at beginWork (react-dom.development.js?61bb:18596)
    at HTMLUnknownElement.callCallback (react-dom.development.js?61bb:188)
    at Object.invokeGuardedCallbackDev (react-dom.development.js?61bb:237)
    at invokeGuardedCallback (react-dom.development.js?61bb:292)
    at beginWork$1 (react-dom.development.js?61bb:23203)
    at performUnitOfWork (react-dom.development.js?61bb:22157)
    at workLoopSync (react-dom.development.js?61bb:22130)
```

Reply:
@noisecapella ^^

Reply:
@gumaerc Can you `yarn install` and try one more time?